pkg/services/org/org.go

```diff
@@ -7,4 +7,5 @@ import (
 type Service interface {
 	GetIDForNewUser(context.Context, GetOrgIDForNewUserCommand) (int64, error)
 	InsertUser(context.Context, *OrgUser) (int64, error)
+	DeleteUser(context.Context, int64) error
```

Comment:
I would suggest renaming in to something like `DeleteOrgUser`

Reply:
the same here `orgService.DeleteUser()` if it does not make sense, I can change it

Reply:
I think we're limiting ourselves to the SQL vocabulary a bit here, wdyt about using `AddUser` and `RemoveUser` here to make it clear that it's not actually creating or removing the user, but adding and removing it from an org?

Reply:
I am ok with both, just was trying to explain my motivation, for me is more important that all the stores and services will follow the same convention

Reply:
This does not delete a user from an organisation (as one could assume by the naming) but clears all raws in the `org_user` table for the specific user. therefore, I think we need a less confusing name. However, naming is hard for me so I don't insist.

Reply:
if it is confusing people I will rename it, I want it to be clear